### PR TITLE
[Hub Generated] Review request for Microsoft.DocumentDB to add version stable/2021-06-15

### PR DESCRIPTION
**Pull request by kavskalyan**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/kavskalyan/azure-rest-api-specs/dev-cosmos-db-Microsoft.DocumentDB-2021-06-15...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by j5lim — APPROVED** [review]
It's promoting a preview api version to a stable one. Approved from ARM side.

**Comment by weidongxu-microsoft** [comment]
Please fix LintDiff.

**Comment by weidongxu-microsoft** [comment]
Loop in @msyyc @ArcturusZhang There is breaking changes report on python and cross-version.

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-06-15/cosmos-db.json [hunk 1]
```diff
@@ -0,0 +1,8807 @@
...
+          },
+          "202": {
+            "description": "Accepted. The offline region operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
```
Comment (weidongxu-microsoft) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-06-15/cosmos-db.json:410]: some error schema seems wrong.

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
@weidongxu-microsoft I approved the Go SDK breaking change. but however, the cross-version breaking changes need the breaking change review board to review and approve

**Comment by weidongxu-microsoft** [comment]
"Cross-Version Breaking Changes" fail on stable to stable version upgrade.

**Review by JeffreyRichter — APPROVED** [review]

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/cosmos-db/resource-manager/readme.md [hunk 2]
```diff
@@ -45,11 +59,25 @@ modelerfour:
   lenient-model-deduplication: true
 ```
 
+
+### Tag: package-2021-06
+
```
Comment (ctaggart) [inline specification/cosmos-db/resource-manager/readme.md:64]: This is duplicate.

File: specification/cosmos-db/resource-manager/readme.md [hunk 3]
```diff
@@ -27,14 +27,28 @@ These are the global settings for the Cosmos-DB API.
 ``` yaml
 title: CosmosDBManagementClient
 openapi-type: arm
-tag: package-2021-05
+tag: package-2021-06
 ```
 
+
+### Tag: package-2021-06
```
Comment (ctaggart) [inline specification/cosmos-db/resource-manager/readme.md:34]: This gets duplicated below.

**Review by ctaggart — COMMENTED** [review]
The `package-2021-06` is duplicated in the readme.

File: specification/cosmos-db/resource-manager/readme.md (hunk 2 above)
Comment (ctaggart) [reply specification/cosmos-db/resource-manager/readme.md:64]: I created https://github.com/Azure/azure-rest-api-specs/issues/14995 for this duplicate tag.

**Pull request merged**